pkg/build/registry/build/strategy.go

```diff
@@ -110,25 +112,39 @@ type detailsStrategy struct {
 
 // Prepares a build for update by only allowing an update to build details.
 // For now, this is the Spec.Revision field
+// We now need to update the build.status.Reason and build.Status.Message
```

Comment:
`// build details currently consists of:  Spec.Revision, Status.Reason, and Status.Message, all of which are updated from within the build pod`